bcipy/task/paradigm/matrix/calibration.py

```diff
@@ -0,0 +1,269 @@
+from typing import List, Tuple
```

Comment:
This class has a LOT of duplication with the RSVP calibration. Now that we have a working task, it would be worth trying to factor out the commonality and determine what is actually unique to matrix. Is it just the display? Maybe we have a parent Calibration class that both RSVP and Matrix inherit from and they override the necessary method. Or maybe the Calibration task takes a display object as a parameter.

Reply:
A ticket was created for this. We'll need to think about how we'd implement row/column vs scp. Many of these methods would change then, but it may make more sense for those two to be separate classes and inherit a common Calibration class as you say. https://www.pivotaltracker.com/story/show/181077145